src/hash-join/managed.cuh

```diff
@@ -0,0 +1,19 @@
+#ifndef MANAGED_CUH

```

Comment:
A copyright header should be added above this line.

Reply:
@kkraus14 it does have the copyright, just not showing in the diff here for some reason

Reply:
Resolved